### PR TITLE
Added eslint to package.json 

### PR DESCRIPTION
**Pull request by ieuanbyers**
Added the eslint dependency and an ignore file to ensure js files aren't linted

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@types/express-session": "^1.17.7",
     "axios": "^1.4.0",
+    "eslint": "^8.45.0",
```
Comment (ccurran01) [inline package.json:20]: Should this be a dependency or a dev dependency?

File: package.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@types/express-session": "^1.17.7",
```
Comment (ccurran01) [inline package.json:18]: Should this be a dependency or a dev dependency?

**Review by ccurran01 — COMMENTED** [review]
eslint typically requires a config file which defines the lining rules you want to enforce. 

The DVSA have an open-source rule set you could follow
https://github.com/dvsa/eslint-config-ts

Alternatively, you could give this one a go?
https://typescript-eslint.io

File: package.json (hunk 1 above)
Comment (ieuanbyers) [reply package.json]: Moved eslint to dev-dependencies

File: package.json (hunk 2 above)
Comment (ieuanbyers) [reply package.json]: Moved @types/express-session to dev-dependencies

**Comment by ieuanbyers** [comment]
I've made those changes you requested and added the .eslintrc used by DVSA @ccurran01

**Comment by ieuanbyers** [comment]
Was going to use the eslint GitHub action. This would be implemented in it's own pull request once these changes are merged.

**Pull request closed without merging**